cmd/porter/credentials.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"github.com/deislabs/porter/pkg/porter"
+	"github.com/deislabs/porter/pkg/printer"
+	"github.com/spf13/cobra"
+)
+
+func buildCredentialsCommand(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "credentials",
+		Aliases: []string{"cred"},
+		Short:   "Credentials commands",
```

Comment:
This needs `Annotations: map[string]string{"group": "resource"}` so that it shows up nice and purdy in the help text when you are ready for that.